src/xkit.cs

```diff
@@ -3963,4 +4077,96 @@ interface NSTextContentStorage : NSTextStorageObserving
 		NSTextRange GetAdjustedRange (NSTextRange textRange, bool forEditingTextSelection);
 	}
 
+	[NoWatch, TV (16,0), iOS (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	interface NSTextList : NSCoding, NSCopying, NSSecureCoding {
+		[Export ("initWithMarkerFormat:options:")]
+#if NET
+		NativeHandle Constructor ([BindAs (typeof (NSTextListMarkerFormats))] NSString format, NSTextListOptions mask);
```

Comment:
This is cool!

Reply:
Breaking change, but this was moved from AppKit to this file :)